honeybadger/contrib/celery.py

```diff
@@ -104,3 +105,10 @@ def tearDowm(self):
         task_postrun.disconnect(self._on_task_postrun)
         if self.report_exceptions:
             task_failure.disconnect(self._on_task_failure)
+
+    # Keep the misspelled method for backward compatibility
+    def tearDowm(self):
```

Comment:
I think we should add `@deprecated("Use tearDown")` to mark the method as deprecated.